src/renderer/hub.html

```
@@ -29,7 +29,42 @@ <h1 class="title">BEACON</h1>
+          <p class="settings-hint">Paste a free API key from <span class="hint-link">console.groq.com</span></p>
+
+          <div class="key-input-row">
+            <input
+              id="api-key-input"
+              type="password"
+              class="key-input"
+              placeholder="gsk_..."
+              spellcheck="false"
+              autocomplete="off"
+            />
+            <button id="save-key-btn" class="save-btn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)